frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -1034,6 +1034,8 @@ func (ds *dispatchState) init() {
 	// the paths we use back to the base image.
 	ds.paths = ds.base.paths
 	ds.workdirSet = ds.base.workdirSet
+	ds.buildArgs = []instructions.KeyValuePairOptional{}
+	ds.buildArgs = append(ds.buildArgs, ds.base.buildArgs...)
```

Comment:
This can be a single-line append, but I'm not sure if we even need the clone at all